modules/claims_api/spec/fixtures/v2/veterans/power_of_attorney/2122a/valid.json

```diff
@@ -14,8 +14,7 @@
       },
       "representative": {
```

Comment:
@acovrig When I sent in the valid.json I get the following:
![Screenshot 2024-03-28 at 1 39 10 PM](https://github.com/department-of-veterans-affairs/vets-api/assets/25069483/2d481caf-5692-43c1-943f-ca3a3e7a0d11)

Reply:
Do you have a rep with that reigistration number and poa code in your local db?